app/views/drug/drug1.html

```diff
@@ -0,0 +1,158 @@
+{% extends "layouts/main.html" %}
+
+{% set pageName="Test" %}
+
+{% block content %}
+<div class="govuk-width-container" style="margin-top: -80px;">
+  <main class="govuk-main-wrapper app-container govuk-body" id="main-content">
+    <div class="govuk-width-container">
+      <div class="govuk-grid-row user-feedback">
+        <div class="govuk-grid-column-full">
+          <div class="govuk-phase-banner user-feedback__banner">
+            <p class="govuk-phase-banner__content">
+              <strong class="govuk-tag govuk-phase-banner__content__tag">
+                Beta
+              </strong>
+              <span class="govuk-phase-banner__text">
+                This is a new service - your <a class="govuk-link user-feedback__link" target="_blank" rel="noopener noreferrer" href="https://forms.office.com/pages/responsepage.aspx?id=KEeHxuZx_kGp4S6MNndq2H4ZAQmHaONKhVwe7eFdwspUMzBFS0JEOU5JOExCQVVZRjNISUMwWDA1SS4u&route=shorturl">feedback (opens in new tab)</a> will help us improve it.
```

Comment:
Probably worth removing the real feedback link from here (and on the other pages)
```suggestion
                This is a new service - your <a class="govuk-link user-feedback__link" target="_blank" rel="noopener noreferrer" href="#">feedback (opens in new tab)</a> will help us improve it.
```